### DISCUSSION TITLE
Improve Medical Jobs

### DISCUSSION THREAD

**Discussion by CPerry631** [FakeFishGames/Barotrauma#12308] [Ideas and suggestions]
I think there is a stark difference between how engaged all the other jobs are in their varied tasks, versus how often a situation actually calls for a doctor. 

As the game goes on, things get harder, but they also become more automated- a doctor making self injection headsets for people feels very strange, like you're cheating yourself out of a job that already is only needed rarely. When people can inject themselves instantly and without attention, even in the midst of combat, without having to be reached by a doctor- how could any player hope to compete?

Similarly it makes more sense to just heal-all at settlement docs rather than try to fix your own team up, since they do your job better and cheaper than you could, which makes doctor professions even less necessary. There's no rationing supplies or making decisions about what is or isn't worth healing yet, you just beeline to a clinic and fix everything for pennies.

As far as I can tell, the vast majority of necessary healing items aren't that hard to apply, and don't call for much medical skill. In my experience playing as and playing with doctors, the common opinion is that they're just boring and have nothing to do 90% of the time. Being the best at healing doesn't really seem to mean a lot when healing is strong as the baseline and has a low barrier of entry. 

I don't know what the best way to help this would be and I don't want to get caught up in specifics of "x or y method should be done", for me it's just about bringing attention to making the experience feel more engaging.

**Comment by Ek-Videogames** [comment]
The uselessness of doctors has been a long discussed issue. Besides their role in fabricating medicine or acting as the dedicated drug mule during EVA - the act of doctoring onboard the ship is made near-pointless by the self administration of medicine as you say.

Possibly you're a new player, as early in the steam beta versions there was a time when the skill requirements were much more lethal for medicines. Long story short that was a whole can of worms, it did make medic 'necessary' but also did make it basically impossible for small crews that didn't have an extra person to be the medic, as well as the unintended effects of making "medicines" the most powerful lethal poisons in the game. _(Morphine/Fentanyl, Antibiotic glue, Broad-spectrum-antibiotics to name a few)_

Without self-administered-care, the time-to-kill for most injuries is also quite high - especially the bleeding inflicted by monster bites or gunshots. Or even if someone is simply unconscious from damage and not bleeding, there is the passive oxygen deprivation timer which makes it very difficult to recover somebody who is down if you don't get to them right away. Due to a combination of those factors, on boats with a more "traditional" medical role - doctors would still find themselves frustrated and ineffectual because most of their patients would just die from the amount of damage they had taken before the doctor could even reach them.

A few ideas I like that I've seen tossed around:
- separate afflictions into "field tended" and "doctor tended" injuries; so simple injuries can be healed with basic first aid knowledge and treatments, or immediate major injuries can be partially treated by once-self or others to regain their health partially & stabilize the injuries somewhat until a doctor arrives or the ship arrives at a station and pays for medical care from the station's doctor.

- increase the use time of various medicines _(maybe also scale it by medical skill so low skill means it takes longer?)_, but increase the amount of time it takes for unconscious crew to die from their afflictions to compensate. So medical care is a bit more methodical and requires more time, as opposed to "hammer the number hotkeys as fast as possible with bandages, blood, and morphine"

**Reply by Ek-Videogames** [reply]
For an example of the 'field tending' idea: say if you had gunshot wounds, it takes a fairly low skill to use a bandage and some morphine to temporarily buy yourself some extra time in the fight & stabilize the bleeding. However this would only be able to heal say, 50% of your lost health. Now: to fully recover you would need to have a doctor perform some different procedure that also comes with a higher skill requirement _(Like maybe adding something a doctor uses to remove bullets from gunshot wounds & shrapnel from explosion wounds, and redressing the wound with bandages "better" when done by the doctor to also improve healing quality.)_.

**Comment by Saevon** [comment]
Field Medics are usually there to patch things up enough to get you to a proper doctor.

Perhaps barotrauma should tried "Unskilled Tending" the same way?
- it suppresses the condition (extreme conditions CANNOT be suppressed) for a short duration
- Now you have to keep self-healing periodically OR get to a proper doctor who can heal you

However this would use up extra resources?  Is it worth healing unskilled but then using the SAME resource when the doctor gets here?  It might encourage crew to risk their lives (make the game less fun) to save resources?
- **Light injuries** just need a doctor to "finish tending" to them (might need a surgical bed) as long as the duration has not run out
- **Complex conditions** can only be suppressed with simple meds for a short time, but need skilled treatment and more expensive meds to cure
- **Severe conditions** can only be suppressed partially (half effects?  lower version? short duration treatment?), and needs a doctor to finish healing

**Comment by G-M-twostay** [comment]
In my medicine overhaul mod I make that many medical operations scales with medic's skill. Also, powerful medicines unlocked in later stages also have way higher skill requirements and aren't all-mighty enough that u just spam it---combing multiple types of medicines is important. In addition, medic's have talents that make their operations more effective. This means that in early game where u r limited to primitive operations everyone can do replace the medic but as game progresses where u encounter more dangers but unlocking new medications, the role of medic becomes more important.
